doc/howto/container_gpu_passthrough_with_docker.md

```diff
@@ -0,0 +1,155 @@
+(container-gpu-passthrough-with-docker)=
+# Passing an NVIDIA dGPU / iGPU to a LXD container with a nested Docker workload
+
+If you have NVIDIA runtime libraries and configuration installed on your host and you want to pass these files to your container, you should add a device with the CDI notation.
```

Comment:
I think you also need the GPU, not only the runtime libraries. ;)

Reply:
fixed